STRUCTURE.md

```diff
@@ -27,23 +27,23 @@
 
 ## Providers
 
-* `Providers` are special React components that are used to provide functionality potentially many layers deep, while avoiding prop-drilling.
```

Comment:
Why has the been changed? 😆

Reply:
Wait, I never touched this XD

Reply:
Oh yes I touched this to remove the `React` from `React.createContext()` (because we don't import React anymore) and it might have linted the file :)